working-open.md

```diff
@@ -0,0 +1,53 @@
+# Working Open
+
+Hypha operates with democratic organizing principles informed by our [values](./values.md). As part of that, we've developed a set of guidelines for how (and exactly what) and why we work in the open.
+
+## Why?
+
+- In order to share and provide opportunities for self- and collective- learning as we work in solidarity with others seeking to create meaningful livelihoods.
+- Not because "default to open" is an ideal in itself, but because it orients us toward negotiating a set of practices around our shared intentions.
+- Yet we recognize openly sharing might prevent frank and necessary discussion beneficial to our members, and so we want to clearly lay out our practices of consent and privacy
+
+# How and what?
+
+- We currently work with a [*in camera*](https://en.wikipedia.org/wiki/In_camera)/redaction request for meeting notes, where at any point members can request to stop taking notes, or redact and correct notes after they have been taken
+
+Legend:  
+👀  view only  
+📝  view and participate  
+
+**Default open to all**:
+
+- Scope: Organization
+  - Meeting notes (all hands, working groups) and final documents (handbook), on GitHub and other platforms 👀
+  - Aggregated financal statements 👀
+- Scope: Project
+  - Project deliverables*, task tracking, meeting notes, and aggregated financial statements 👀
+
+* *This would include source repositories and code (e.g., on GitHub) and other process materials based on consensus on licensing and usage constraints.*
+
+**Default open to members**:
+
+- Scope: Organization
+  - Meeting notes and documents 📝 
+  - Task tracking, project management (PM), and decision making (DM) tools 📝 
+  - Finances (budget, expenses, and cash flow projections) and audit records 👀
+  - Potential opportunities and leads 📝 
+- Scope: Project
+  - Project proposal(s), statement(s) of work (in-progress/signed), in-progres deliverables, and finances (budget, expenses, and cash flow projections) 👀
+
+**Default closed to specific working groups and project members**:
+
+- Scope: Organization
+  - Full personal/financial information of members (e.g., home addresses, banking information, and social insurance numbers)
```

Comment:
```suggestion
  - Full personal/financial information of members (e.g. home addresses, banking information, and social insurance numbers)
```

Reply:
I favour commas after e.g./i.e., it isn't a typo but a rather a distinct usage, see discussion: https://www.dailywritingtips.com/comma-after-i-e-and-e-g/ (I'd also say it fits in this text)

Reply:
Not saying this one is a typo, but I think without the comma is more common and so more likely to be consistently used. Same with 1 or 2 spaces after periods, etc. If you prefer having the comma we can aim for that in all our documents, but I definitely feel not having it is likely to stay consistent.

Reply:
We could use [`vale`](https://errata-ai.github.io/vale/) and [dangerCI gem](https://danger.systems/guides/what_does_danger_do.html) to encode our rules and do some of these evolving checks. Demo: https://github.com/patcon/dangerfile-test/pull/1

Style rule: https://github.com/patcon/dangerfile-test/blob/master/ci/vale/styles/HyphaCoop/Eg.yml

Reply:
I'm pretty indifferent about which, though I'll need to get used to using commas. But I'm fine with whatever conventions as long as we're on same page :)

(hence my appreciating the bot auto-correcting)

Reply:
Commas after "For example" (e.g.) and "Further" (i.e.) tends to be grammatically correct when used without abbreviations. In my experience, but this is based on a lot of academic reading I recognize, commas are the more frequent usage.